Makefile

```diff
@@ -0,0 +1,14 @@
+generate:
+	cd api; buf generate; mv ../pkg/microservice.swagger.json ../static/swagger-ui/swagger.yaml
```

Comment:
It seems that if the host machine is missing some prerequisite tools this step fails. What would you do to improve the situation here?